### DISCUSSION TITLE
Best way to represent Matlab script

### DISCUSSION THREAD

**Discussion by majdgh92** [typst/typst#2395] [Q&A] [answered]
Hello,
i have a lot of MATLAB scripts that i would like to add them in the papers. is there any preferred way to add these code blocks?
Thanks in Advance.

**Comment by emilyyyylime** [comment, marked as answer]
You want to show the code itself? Isn't `raw` what you're elooking for?
`````typ
```matlab
... code ...
```
`````

**Reply by majdgh92** [reply]
i have found your solution a couple of minutes ago and used it. so i closed the discussion but i am not sure how where you able to find it and answer it. I thank you very much anyway.

**Reply by Thumuss** [reply]
From [Typst docs](https://typst.app/docs/reference/text/raw/)
> [...] This is typically used to embed computer code into your document.
